jscomp/bsb/templates/basic-reason/bsconfig.json

```diff
@@ -16,6 +16,5 @@
   "warnings": {
     "error" : "+101"
   },
-  "namespace": true,
-  "refmt": 3
```

Comment:
What happens to this config option? I think it should just be ignored for now -- no need to introduce a breaking change.

Reply:
I thought it was already ignored and could safely be removed? I don't have this setting in any of the bsconfig.json files in our project.

Reply:
Sorry I did not mean to imply that it is not ignored. It was just a thought, to check that it is indeed the case.